xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd

```diff
@@ -118,18 +132,6 @@ Rail transport, Roads and Road transport
 		</xsd:sequence>
 	</xsd:group>
 	<!-- =====GROUPING=============================================== -->
-	<xsd:complexType name="groupOfEntitiesInFrame_RelStructure">
```

Comment:
So where did this go?

Reply:
line 67

Reply:
Moved to netex_framework/netex_genericFramework/netex_grouping_version.xsd:67
(except that, contents of v1.0 and v.1.1 seems to be the same)